Source/JavaScriptCore/b3/air/AirArg.h

```diff
@@ -495,6 +497,8 @@ class Arg {
 
     static Arg imm(int64_t value)
     {
+        if constexpr (is32Bit())
```

Comment:
I am not sure that changing the semantics of imm per-platform is a good idea. Maybe we should have a ptrImm instead?

Reply:
Actually, maybe just a smallImm? Hmm, I am not sure now